src/Init/Data/Option/Basic.lean

```diff
@@ -89,6 +89,11 @@ Implementation of `OrElse`'s `<|>` syntax for `Option`.
 instance : OrElse (Option α) where
   orElse := Option.orElse
 
+/-- If the first argument is `some a`, returns `some a`, otherwise returns the second argument. -/
```

Comment:
Could the doc-string explicitly reference `orElse` (and vice versa). I worry still about people being confused about which one they should be using.